editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -581,7 +581,7 @@ void TileSetEditor::_on_textures_added(const PoolStringArray &p_paths) {
 	texture_list->select(texture_list->get_item_count() - 1);
 	_on_texture_list_selected(texture_list->get_item_count() - 1);
 	if (invalid_count > 0) {
-		err_dialog->set_text(String::num(invalid_count, 0) + TTR(" file(s) was not added because was already on the list."));
+		err_dialog->set_text(TTR("%s file(s) was not added because was already on the list."), String::num(invalid_count, 0));
```

Comment:
Needs `vformat` too. I'd suggest to use plural by default too as it's bound to be 1 or more, so "%s file(s) were skipped because they were already on the list."